### DISCUSSION TITLE
About the difference between the mysql version of 1panel professional version and the community version (the professional version does not provide version 8.2)

### DISCUSSION THREAD

**Discussion by dangks** [1Panel-dev/1Panel#7297] [Q&A]
I recently started the 1panel professional version. During the process of building the service, I found that the MySQL installation provided by the app store does not have version 8.2, and I have been using version 8.2 before. According to my actual experience, version 8.2 saves more memory than version 8.4. Why is version 8.2 no longer available? Is there any problem?

![Prtscr_2024-12-09_14-18-42](https://github.com/user-attachments/assets/94955dc8-0dc3-4cc5-8b39-53c43e7f94ee)

![Prtscr_2024-12-09_14-49-17](https://github.com/user-attachments/assets/54fd56d0-9e7b-4caa-a783-065183cf693e)

![Prtscr_2024-12-09_14-24-24](https://github.com/user-attachments/assets/537e63ff-e29f-4709-b5bf-b7e303ecbada)

**Comment by wanghe-fit2cloud** [comment]
Version 8.4 is an upgrade from 8.2. Therefore, we align with MySQL’s official practice by retaining four LTS versions: 5.6, 5.7, 8.0, and 8.4.

**Reply by dangks** [reply]
Thanks for your reply. But I think as one of the many individual users, pursuing good server performance is what we want. At present, your community version is still providing version 8.2, and I think the professional version should also continue to provide it.

**Reply by wanghe-fit2cloud** [reply]
这是因为你的社区版本之前安装过 MySQL 8.2。如果再次安装，系统会默认选择本地已存在的安装包进行安装。
如果必须使用 MySQL 8.2，也可以在安装 8.4 时手动将镜像版本修改为 8.2，如图：
![image](https://github.com/user-attachments/assets/1f5a1c09-a83b-438b-bb90-074db1b7a6dd)

**Reply by dangks** [reply]
Got it, thanks.
